src/system_info/system_info_sim_ivi.cc

```diff
@@ -23,11 +14,18 @@ SysInfoSim::SysInfoSim()
       msin_(""),
       spn_(""),
       conn_(NULL) {
-  InitDbusConnection();
+  conn_ = system_info::GetDbusConnection();
+  if (!conn_)
+    return;
+  modem_path_ = system_info::OfonoGetModemPath(conn_);
```

Comment:
`system_info::OfonoGetModemPath()` seems will be called in multiple places. To saving the the dbus calls, I'd suggest to rewrite it to return a static result if it is ever be called. @zolkis any comment?